src/main/java/com/google/jenkins/plugins/computeengine/CleanLostNodesWork.java

```diff
@@ -40,17 +43,28 @@
 @Symbol("cleanLostNodesWork")
 public class CleanLostNodesWork extends PeriodicWork {
     protected final Logger logger = Logger.getLogger(getClass().getName());
-    public static final String NODE_IN_USE_LABEL_KEY = "jenkins_node_in_use";
+    public static final String NODE_IN_USE_LABEL_KEY = "jenkins_node_last_refresh";
     public static final long RECURRENCE_PERIOD = Long.parseLong(
-            System.getProperty("jenkins.cloud.gcp.cleanLostNodesWork.recurrencePeriod", String.valueOf(HOUR)));
+            System.getProperty(CleanLostNodesWork.class.getName() + ".recurrencePeriod", String.valueOf(HOUR)));
     public static final int LOST_MULTIPLIER = 3;
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy_MM_dd'T'HH_mm_ss_SSSX");
```

Comment:
IIRC there is a set of predefined formatters for ISO formats.

Reply:
So then why call `toUpper` / `toLower`? Just use lowercase letters in your format to begin with.

Reply:
that is more correct i think,
fixed by `yyyy_MM_dd't'HH_mm_ss_SSS'utc'` the example value like `2025_01_07t20_34_48_812utc`.